packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -81,8 +95,12 @@ interface Item {
 	value: string;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function DataRow(label: string, id: string | undefined, columnProps: any): React.ReactElement {
+function DataRow(
+	content: React.ReactFragment,
```

Comment:
This should probably be:
```suggestion
	content: React.ReactElement,
```